func/rebuild.sh

```diff
@@ -98,6 +98,7 @@ rebuild_user_conf() {
         mkdir -p $HOMEDIR/$user/tmp
         chmod 751 $HOMEDIR/$user/conf/web
         chmod 751 $HOMEDIR/$user/web
+        chmod 755 $HOMEDIR/$user/web/public_*html
```

Comment:
No need for this, public _html is only present in domain folder

Reply:
You are correct. In the v-rebuild-web-domains the correct permissions are set.